content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-12.md

```diff
@@ -44,7 +44,7 @@ This page explains how to upgrade a Kubernetes cluster created with `kubeadm` fr
     apt-mark hold kubeadm
     {{% /tab %}}
     {{% tab name="CentOS, RHEL or Fedora" %}}
-    yum upgrade -y kubeadm --disableexcludes=kubernetes
+    yum upgrade -y kubeadm-1.12.5 --disableexcludes=kubernetes
```

Comment:
instead of pinning a PATCH version, this can be done:

```
# replace "x" with the latest patch version
yum upgrade -y kubeadm-1.12.x --disableexcludes=kubernetes
```

similar has to be done for this section:
> Upgrade the Kubernetes package version on each $NODE node by running the Linux package manager for your distribution:

Reply:
Finished! Thank you for your reminder!